app/views/people/membership/verify/show.html.haml

```diff
@@ -21,7 +21,7 @@
       #content
         - if person
           .member-img
-            = image_tag(person.picture_url)
+            = image_tag(upload(person, :picture))
```

Comment:
Gehört das wirklich in diesen PR? Warum musstest du das ändern?

Reply:
ah das ist wohl beim rebasen reingerutscht :raised_eyebrow:

Reply:
Also: nein, eigentlich sollte dieser PR nur https://github.com/hitobito/hitobito_skv/pull/39/commits/2fb3d2ff25ee737fbb5f87f6da1523b10a98f46c umfassen